pkg/fakerp/fakerp.go

```diff
@@ -41,13 +41,23 @@ func CreateOrUpdate(ctx context.Context, oc *v20180930preview.OpenShiftManagedCl
 		return nil, err
 	}
 
+	// determine the paths where various configs should be written on disk
+	configBlob := os.Getenv("CONFIG_BLOB_PATH")
+	helpers := os.Getenv("HELPERS_PATH")
```

Comment:
Why do you need these changes?

Reply:
I needed these changes because
1) Allow a developer to change where the config blob path and helpers are written to disk. Without that, I found that e2e helpers and the config blob were being written to the various e2e suite root directories. For key rotation tests, I wanted each rotation to cause the previous helpers and config blob files to be overwritten.
2) These values were hardcoded in the code. Those values still remain the default. These env vars will help to tweak those locations on an ad-hoc basis.